sdk/database/dbplugin/v5/grpc_server.go

```diff
@@ -69,6 +78,10 @@ func (g gRPCServer) getOrCreateDatabase(ctx context.Context) (Database, error) {
 
 // getDatabaseInternal returns the database but does not hold a lock
 func (g gRPCServer) getDatabaseInternal(ctx context.Context) (Database, error) {
+	if !g.multiplexingSupport && g.singleImpl != nil {
```

Comment:
Is the `multiplexingSupport` needed, or can we base the mode off the presence/absence of `g.singleImpl`?

Reply:
Yeah, that is what I had initially. multiplexingSupport isn't really needed. I added it thinking it would make things more clear. If your vote is to remove it then I am good with that too.